CorsixTH/Lua/research_department.lua

```diff
@@ -343,8 +343,8 @@ function ResearchDepartment:addResearchPoints(points, autopsy_room)
   ---------------------- An autopsy has been done ---------------------------
```

Comment:
An aside, this function badly needs to be split into two functions, one for autospy research and one for general research.